iabtcf-decoder/src/main/java/com/iabtcf/decoder/TCStringV2.java

```diff
@@ -263,15 +264,15 @@ public Instant getCreated() {
         if (cache.add(CORE_CREATED)) {
             consentRecordCreated = Instant.ofEpochMilli(bbv.readBits36(CORE_CREATED) * 100);
         }
-        return consentRecordCreated;
+        return consentRecordCreated.truncatedTo(ChronoUnit.DAYS);
```

Comment:
i'm minus one on this. we should return whatever was encoded.

Reply:
Yeah, I was not sure myself. I think it's better to return whatever is encoded. I will make the change.

Reply:
@laktech Ok done pushing my changes.